docs/docs/examples/bitbucket/.pages

```diff
@@ -1,5 +1,6 @@
 nav:
-    - Elasticsearch : BITBUCKET_ELASTICSEARCH.md
+    - OpenSearch    : BITBUCKET_OPENSEARCH.md
+    - Elasticsearch (Deprecated) : BITBUCKET_ELASTICSEARCH.md
```

Comment:
We should add warning panel to the `BITBUCKET_ELASTICSEARCH.md` page and link the OpenSearch page.